### PR TITLE
Update What's New to Include information on Timezone and Applicability changes in transformations

### PR DESCRIPTION
**Pull request by codeincarnate**
This PR updates the what's new to include information on https://github.com/grafana/grafana/pull/76316 and https://github.com/grafana/grafana/pull/76373. These changes include adding better timezone support when formatting times, and a new system to de-emphasize transformations that can't work on given data.

### CODE REVIEW

**Comment by codeincarnate** [comment]
This PR for the main docs PR at  #75909

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 1]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
 
 Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).
 
+### Detect Unusable Transformations
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:189]:
```suggestion
### Detect unusable transformations
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 2]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+<!-- Kyle Cunningham -->
+
+_Available behind a feature flag in Grafana Open Source and Enterprise_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:193]: We should talk about this in terms of a release stage. Does this align with [experimental, private preview, or public preview](https://grafana.com/docs/release-life-cycle/)?

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 3]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+We've added initial support to detect situations in which various transformations won't work appropriately based on current data. Previously, in selecting transformations it would require a process of trial and error or otherwise knowing how a given transformation worked beforehand to configure them correctly. Transformations that we've detected can't be used will be shaded in the interface to indicate they can't be used along with a helpful message explaining why.
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:195]:
```suggestion
We've added initial support to detect situations in which various transformations won't work appropriately based on current data. Previously, selecting the appropriate transformation and configuring it correctly required a process of trial and error or already knowing how a given transformation worked. Now, transformations that we've detected can't be used are shaded in the interface to indicate this, along with a helpful message explaining why.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 4]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+{{< figure src="/media/docs/grafana/transformations/disabled-transformation.png" caption="Transformation that has been disabled because it doesn't have the necessary data" >}}
+
+For users with the `transformationsRedesign` feature flag set, you'll be able to access this functionality right away. For everyone who would like to try it, this feature flag can be enabled in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:199]:
```suggestion
If you have the `transformationsRedesign` feature flag set, you'll be able to access this functionality right away. If you'd like to try it, enable this feature flag in your [Grafana configuration](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/#feature_toggles).
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 5]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+### Extended time zone support in Format time and Convert field type transformations
+
+<!-- Kyle Cunningham -->
+
+_Generally available in Grafana Open Source and Enterprise_
+
+Support has been added to set timezones manually when formatting times as strings using the *Format time* and *Convert field type*. This allows times to be formatted relative to any timezone across the globe!
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:207]:
```suggestion
We've added support for setting timezones manually when formatting times as strings using the **Format time** and **Convert field type** transformations. This allows times to be formatted relative to any timezone across the globe.
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 6]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+{{< figure src="/media/docs/grafana/transformations/format-timezone.png" caption="Timezone support in the format time transformation" >}}
+
 > > > > > > > docs/whats-new-v10.2
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:211]: Not sure where this came from
```suggestion
```

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 7]
```diff
@@ -186,4 +186,26 @@ With the new **Format string** transformation, you can manipulate string fields
...
+
+### Extended time zone support in Format time and Convert field type transformations
+
+<!-- Kyle Cunningham -->
+
+_Generally available in Grafana Open Source and Enterprise_
+
+Support has been added to set timezones manually when formatting times as strings using the *Format time* and *Convert field type*. This allows times to be formatted relative to any timezone across the globe!
+
+{{< figure src="/media/docs/grafana/transformations/format-timezone.png" caption="Timezone support in the format time transformation" >}}
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:209]:
```suggestion
{{< figure src="/media/docs/grafana/transformations/format-timezone.png" caption="Timezone support in the Format time transformation" >}}
```

**Review by imatwawana — COMMENTED** [review]
Made some wording and style edits and left a question re: release stage of one of the items.

**Comment by codeincarnate** [comment]
Thanks for the feedback @imatwawana! I'll get this updated today 😄

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 8]
```diff
@@ -251,6 +251,28 @@ With the new **Format string** transformation, you can manipulate string fields
 
 Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).
 
+### Detect unusable transformations
+
+<!-- Kyle Cunningham -->
+
+_Available in public preview in all editions of Grafana_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:258]: I'll add this separately to the Cloud what's new and tag you for review.
```suggestion
_Available in public preview in all Grafana Open Source and Enterprise_
```

**Review by imatwawana — APPROVED** [review]
Apart from one small thing, this is good to go!

File: docs/sources/whatsnew/whats-new-in-v10-2.md [hunk 9]
```diff
@@ -251,6 +251,28 @@ With the new **Format string** transformation, you can manipulate string fields
 
 Learn more in the [Format string documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data/#format-string).
 
+### Detect unusable transformations
+
+<!-- Kyle Cunningham -->
+
+_Available in public preview in all Grafana Open Source and Enterprise_
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-in-v10-2.md:258]:
```suggestion
_Available in public preview in Grafana Open Source and Enterprise_
```

**Comment by codeincarnate** [comment]
Thanks @imatwawana 🙏

**Pull request merged**